services/management/dbaas/psmdb_cluster_service.go

```diff
@@ -89,7 +89,9 @@ func (s PSMDBClusterService) ListPSMDBClusters(ctx context.Context, req *dbaasv1
 			},
 			State: psmdbStates()[c.State],
 			Operation: &dbaasv1beta1.RunningOperation{
-				Message: message,
+				TotalSteps:    c.Operation.TotalSteps,
+				FinishedSteps: c.Operation.FinishedSteps,
+				Message:       message,
```

Comment:
let's remove https://github.com/percona/pmm-managed/pull/623/files#diff-7a56b03a49f9c4fd843c3edfc194110f786609bba47b3ec694b844ec0f3aff8cR78 check and use `c.Operation.Message` directly

Reply:
@BupycHuk I removed the check. Did you mean it like that?

Reply:
yes, thank you